.travis.yml

```diff
@@ -45,6 +46,7 @@ matrix:
     env: USE_CMAKE=YES
 
 script:
+  - if [ ${TRAVIS_OS_NAME} = 'linux' ]; then KWStyle -R -v -gcc -d src; fi
```

Comment:
I see. I then agree that, at least for now, the tests folder need not be checked. If you are interested in proposing fixes for the existing issues, feel free to have a lax configuration for now and it can be incrementally tightened as times goes on.